test/test_spectral_ops.py

```diff
@@ -5,11 +5,15 @@
 from itertools import product
 
 from torch.testing._internal.common_utils import \
-    (TestCase, run_tests, TEST_NUMPY, TEST_LIBROSA)
+    (TestCase, run_tests, TEST_NUMPY, TEST_LIBROSA, _assertGradAndGradgradChecks)
```

Comment:
For autograd checks update method_tests in torch/testing/_internal/common_methods_invocations.py. 

I'm not sure if method_tests requires tests have method variants (it shouldn't), but if it does you can add method variants for these functions and not document them.

Reply:
I added a simple test to `common_method_invocations.py`. I also had to modify `test_autograd.py` to support qualified names like `'fft.fft'`.

Reply:
You could also use the ugly torch name, like torch.fft_fft.

Reply:
That doesn't exis in the torch namespace, it's only in `torch._C._fft.fft_fft`.

Reply:
Oh you're right, sorry. It's only in the C++ namespace that we still have names like fft_fft.